app/page/template/partials/template-message.htm

```diff
@@ -86,9 +86,9 @@
     </div>
   <!-- /ko -->
       
-  <div class="message-body">
+  <div class="message-body" data-bind="attr: {'title': message.ephemeral_caption()}">
```

Comment:
Did you discuss this change with design?

Reply:
This is the intended new behaviour.